src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/Forwarder.vue

```diff
@@ -206,9 +206,6 @@ export default {
 		display: flex;
 		justify-content: space-between;
 		padding: 12px;
-		button {
-			height: 44px;
-		}
```

Comment:
In the same commit that fixes the options? You want me to cry, don't you? :-P

Reply:
Yes, sorry. This was meant to be part of #7704 but I stashed the file as my brain was linking it to the forward thing. And I was too lazy to fix that afterwards.